src/main/kotlin/com/terraformation/backend/search/table/SpeciesTable.kt

```diff
@@ -23,21 +25,26 @@ class SpeciesTable(tables: SearchTables) : SearchTable() {
   override val sublists: List<SublistField> by lazy {
     with(tables) {
       listOf(
-          speciesEcosystemTypes.asMultiValueSublist(
-              "ecosystemTypes", SPECIES.ID.eq(SPECIES_ECOSYSTEM_TYPES.SPECIES_ID)),
+          inventories.asSingleValueSublist(
+              "inventory",
+              SPECIES.ORGANIZATION_ID.eq(INVENTORIES.ORGANIZATION_ID)
+                  .and(SPECIES.ID.eq(INVENTORIES.SPECIES_ID)),
+              isRequired = false),
           nurserySpeciesProjects.asMultiValueSublist(
               "nurseryProjects", SPECIES.ID.eq(SPECIES_PROJECTS.SPECIES_ID)),
           organizations.asSingleValueSublist(
               "organization", SPECIES.ORGANIZATION_ID.eq(ORGANIZATIONS.ID)),
-          speciesProblems.asMultiValueSublist(
-              "problems", SPECIES.ID.eq(SPECIES_PROBLEMS.SPECIES_ID)),
+          speciesEcosystemTypes.asMultiValueSublist(
+              "ecosystemTypes", SPECIES.ID.eq(SPECIES_ECOSYSTEM_TYPES.SPECIES_ID)),
           speciesGrowthForms.asMultiValueSublist(
               "growthForms", SPECIES.ID.eq(SPECIES_GROWTH_FORMS.SPECIES_ID)),
-          inventories.asSingleValueSublist(
-              "inventory",
-              SPECIES.ORGANIZATION_ID.eq(INVENTORIES.ORGANIZATION_ID)
-                  .and(SPECIES.ID.eq(INVENTORIES.SPECIES_ID)),
-              isRequired = false),
+          speciesPlantMaterialSourcingMethods.asMultiValueSublist(
+              "plantMaterialSourcingMethods",
+              SPECIES.ID.eq(SPECIES_PLANT_MATERIAL_SOURCING_METHODS.SPECIES_ID)),
+          speciesProblems.asMultiValueSublist(
+              "problems", SPECIES.ID.eq(SPECIES_PROBLEMS.SPECIES_ID)),
+          speciesSuccessionalGroups.asMultiValueSublist(
+              "successionalGroups", SPECIES.ID.eq(SPECIES_SUCCESSIONAL_GROUPS.SPECIES_ID)),
```

Comment:
The addition of `speciesPlantMaterialSourcingMethods` and `speciesSuccessionalGroups` are the only functional changes, the rest is moving things to alphabetical order.

Reply:
Yeah, this is a fun one because there are two reasonable alphabetical orders: the table's property name in `SearchTables` and the sublist name.